src/net.cpp

```diff
@@ -2102,11 +2102,18 @@ void CConnman::ThreadOpenMasternodeConnections()
 
     auto& chainParams = Params();
 
+    bool didConnect = false;
     while (!interruptNet)
     {
-        if (!interruptNet.sleep_for(std::chrono::milliseconds(1000)))
+        int sleepTime = 1000;
+        if (didConnect) {
+            sleepTime = 100;
+        }
```

Comment:
could be, `sleepTime = didConnect ? 100 : 1000;`

Reply:
Nah, I'm not a fan of the ternary operator. It's horrible to read and understand.